cob_teleop/ros/src/cob_teleop_v1.cpp

```diff
@@ -540,27 +540,33 @@ void TeleopCOB::joy_cb(const sensor_msgs::Joy::ConstPtr &joy_msg)
 	{
 		ros::ServiceClient client_init_base = n_.serviceClient<cob_srvs::Trigger>("/base_controller/init");
 	
-		ROS_INFO("Init base");
+		ROS_INFO("Initializing base...");
 		cob_srvs::Trigger srv = cob_srvs::Trigger();
 		if (client_init_base.call(srv))
 		{
-			ROS_INFO("Base init successfully");
+			if (!srv.response.success.data)
+				ROS_ERROR("Failed to initialize base.");
+			else
+				ROS_INFO("Base successfully initialized.");
 		}
 		else
 		{
-			ROS_ERROR("Failed to call service /base_controller/init");
+			ROS_ERROR("Failed to call service /base_controller/init.");
```

Comment:
@ipa-mig-tk @ipa-mig 
It's the same hardcoded namespace in here, too (see [cob_robots#204](https://github.com/ipa320/cob_driver/pull/204))
Please change this!